message-index/messages/GHC-24180/index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Too few arguments to type operator
```

Comment:
```suggestion
title: Too few arguments to infix operator
```

I wasn't quite sure how to suggest this. Either I think it needs to be clear from the title that this is for *binary* operators (and therefore this is an infix problem), or that the summary should suggest there are *multiple* operands required

edit: after reading the examples and realising that this is for *type level operators*, I think the above is still valid but I think some emphasis that this is a *type level problem* rather than a *value level problem*